Add header branch files and folders

luisrpavanelli opened the pull request:
Hello, in this work step I am adding the folders and all the structural files counting on the Html and Css files to make the portfolio project

luisrpavanelli commented:
Hello, in this work step I am adding the folders and all the structural files counting on the Html and Css files to make the portfolio project

iambenkis reviewed (changes requested):
Hi @luisrpavanelli , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 

- [ ] You have submitted a project that does not have a README file , kindly add the [microverse README template](https://github.com/microverseinc/readme-template) and customize it according to your project :+1: 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me `@iambenkis` in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

luisrpavanelli commented:
Hello, in this work step I am adding the folders and all the structural files counting on the Html and Css files to make the portfolio project

Lizdev-05 commented on index.html:
```diff
@@ -0,0 +1,56 @@
+    <!-- Headline Section-->
+    <section class="headline">
+        <div class="container" id="welcometext">
+            <h1>Hey there. I'm Luis Renan Pavanello</h1>
```
Great job so far 👍. I will like to suggest that you use just your firstname so that your work will be inline with the Figma design.

| Your Design | Figma Design |
| :---:   | :---: | 
|<img width="351" alt="image" src="https://user-images.githubusercontent.com/73854999/219856790-59af33bf-773f-4a01-b2f9-1b90e6bd9566.png"> | <img width="351" alt="image" src="https://user-images.githubusercontent.com/73854999/219856804-580a6d27-7aeb-4059-85ca-577caec4aaa4.png">|

Lizdev-05 commented on index.html:
```diff
@@ -0,0 +1,56 @@
+    <header>
+        <a class="logo" href="#">Welcome</a>
+        <nav>
+            <img id="menuicon" src="./icons/hamburguermenu.png" atl="Menu" />
+        </nav>
+    </header>
+    <!-- Headline Section-->
+    <section class="headline">
+        <div class="container" id="welcometext">
+            <h1>Hey there. I'm Luis Renan Pavanello</h1>
+            <h2 id="highlight">I'm a software developer</h2>
+        </div>
+        <div class="container">
+            <p>
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.
+            </p>
+        </div>
+        <div class="container">
+            <ul>
+                <li class="socialmediaicons">
+                    <a href="https://twitter.com/"><img src="./icons/socialmedia/twitter.png" alt="Twitter"></a>
+                </li>
+                <li class="socialmediaicons">
+                    <a href="https://www.linkedin.com/"><img src="./icons/socialmedia/linkedin.png" alt="LinkedIn"></a>
+                </li>
+                <li class="socialmediaicons">
+                    <a href="https://medium.com/"><img src="./icons/socialmedia/medium.png" alt="Medium"></a>
+                </li>
+                <li class="socialmediaicons">
+                    <a href="https://github.com/"><img src="./icons/socialmedia/github.png" alt="GitHub"></a>
+                </li>
+                <li class="socialmediaicons">
+                    <a href="https://www.angellist.com/"><img src="./icons/socialmedia/angellist.png" alt="Angel List"></a>
+                </li>
+            </ul>
+        </div>
+        <div class="container">
+            <a href="#nextSection"><img src="./icons/continue_button.png" alt="Continue"></a>
+        </div>
```
Kindly add the backgroung images that we have on the Figma design. This is part of the requiremements for this project. ☺️ 🟢 

| Your Design | Figma Design |
| :---:   | :---: | 
|<img width="375" alt="image" src="https://user-images.githubusercontent.com/73854999/219856924-2c378db1-3464-4907-8357-f9f826c68787.png"> |<img width="375" alt="image" src="https://user-images.githubusercontent.com/73854999/219856955-a8d966b5-8260-4747-b38e-d2aebb4c98a5.png"> |

Lizdev-05 reviewed (changes requested):
Hi @luisrpavanelli, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] Good use of GitHub flow ✔️
- [ ] Meaningful commit messages ✔️
- [ ] No linter error ✔️
- [ ] Brilliant implementation ✔️

## Required Changes ♻️

- [ ] I noticed that you added the README file as suggested by the previous reviewer, but you added it to the base branch. Next time, kindly make your changes in the current branch and if at all you make any changes in the base branch, kundly keep the current branch updated. 👍 🟢 😉 
- [ ] In your README file, the MIT link is broken, kindly set it up by using the template on the [GItHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) 😊 ☺️ 
- [ ] Kindly make sure your pull request is always descriptive. You can have something like : In this pmilestone, I achieved the following...
  - Created the toolbar that housed the logo and nav
  - Created the intro section
  - Styled the project with css etc

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

luisrpavanelli commented:
Thank you so much, I will use that suggest: In this pmilestone, I achieved the following...
Created the toolbar that housed the logo and nav
Created the intro section
Styled the project with css etc

Favourezeugwa commented on style.css:
```diff
@@ -0,0 +1,83 @@
+  justify-content: space-between;
+}
+
+.headline {
+  text-align: center;
+  max-height: 725px;
+  background-image: url("./icons/header-mobile-bg.png");
+  background-size: cover;
+  background-repeat: no-repeat;
+  padding: 64px 24px;
+  margin: auto;
+  display: flex;
+  gap: 58px;
+  flex-direction: column;
+  justify-content: center;
+}
```
- Great Job adding the background image as suggested by the previous reviewer. But there is a little bit of an issue. The background image seems to be too large on the mobile view. Can you reduce(edit) the background image a bit to suit your mobile screen? Or download an image for the preferred screen. 🤗 

This is what I can see compared to Figma 👇🏽 

| Your Design  | Figma Design |
| :---:   | :---: | 
| <img width="386" alt="Screenshot 2023-02-18 at 06 26 22" src="https://user-images.githubusercontent.com/99246359/219865919-e868d041-58fe-4a06-b8ab-f28862f37fdb.png"> |<img width="341" alt="Screenshot 2023-02-18 at 06 25 59" src="https://user-images.githubusercontent.com/99246359/219865956-48a7a9ed-c115-410d-b02f-c129a4c21989.png">  |

Favourezeugwa reviewed (changes requested):
## STATUS: REQUIRES CHANGES 🚦 ⚠️ ♻️ 

Hi @luisrpavanelli  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


![close lady](https://user-images.githubusercontent.com/99246359/207367678-ef4e9b86-75b7-4d8e-b9b5-899f2e87c934.gif)

## Highlights 💯 
- No Linter errors ✔️
- Descriptive Pull request summary✔️
- Good commit messages ✔️
- Professional Readme file ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

luisrpavanelli commented:
I changed it to auto adjust to mobile I think now it should be correct

amon-cofie commented on style.css:
```diff
@@ -0,0 +1,83 @@
+  justify-content: space-between;
+}
+
+.headline {
+  text-align: center;
+  max-height: 725px;
+  background-image: url("./icons/header-mobile-bg.png");
+  background-size: 100%;
+  background-repeat: no-repeat;
+  padding: 64px 24px;
+  margin: auto;
+  display: flex;
+  gap: 58px;
+  flex-direction: column;
+  justify-content: center;
+}
```
- [ ] I can see that you did your best to implement the change required by the previous reviewer. However, unfortunately, a new bug was introduced in your UI. Please try to fix it. I'll leave a few suggestions but expect that you test and make sure that it works before resubmission.
```
background-size: cover;
background-position: center;
```
Your design now: 
![image](https://user-images.githubusercontent.com/107795045/219869480-27e8db8f-3deb-488e-ba23-907b13c858f6.png)

The figma template:
![image](https://user-images.githubusercontent.com/107795045/219869531-710f63cf-788b-450c-bebc-63f8767f77f7.png)

amon-cofie reviewed (changes requested):
Hi @luisrpavanelli , 

Great work on making the changes requested by a previous reviewers 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet and I believe you can make them. 💪 

<img src="https://media3.giphy.com/media/m3864rBwwBTKMAbpn3/giphy.gif"/>

 ## Highlights
No Linter errors ✔️
Descriptive Pull request summary✔️
Good commit messages ✔️
Professional Readme file ✔️

 ## Required Changes ♻️

I can see that you took out the README.md file and I would have loved to know why but I suggest that you add it on your next submission as it is crucial to the approval of you project and it is the documentation of your project.

_Check the comments under the review._

## Optional suggestions

You can contact me on slack or zoom and if it is feasible I can answer any question you may have 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

luisrpavanelli commented:
I changed the settings, background-size: cover; background-position: center; as requested.

algerina reviewed (approved):
Hi @luisrpavanelli ,

Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:

Your project is complete!  It's time to merge it :shipit:

<img src="https://media4.giphy.com/media/LlYjGi9rErbLjyaMNr/giphy.gif"/>

 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

luisrpavanelli commented:
In this milestone, I achieved the following...

- added images and logo
- added parts of code in html file
- added code parts in css file
- added links and buttons

sections were included

- recent works section: multi-post stories
- section about me
- languages section
- frameworks section
- skills section

topeogunleye reviewed (commented):
Hi @luisrpavanelli, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a used PR for a new project. Kindly create a new PR for new projects.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

ouasamine reviewed (commented):
Hi @luisrpavanelli  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted the wrong PR for the project, please make sure to submit the right PR for this project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
